src/vic-stdlib.c

```diff
@@ -44,6 +46,33 @@ void vic_get(void)
     }
 }
 
+void vic_list(void)
+{
+    char option = '\0';
+    char *args_format = "-%c ";
+
+    if (vic_args(args_format, &option) != 1) {
```

Comment:
@Adman Do you mind if I make two functions: lsv, lsf (or rather mrshu's version suggestion)?